src/renderer/MainContent.tsx

```diff
@@ -25,73 +22,35 @@ const MainContent = () => {
         const sliderCount = await getEqualizerSliderCount();
         const newIndices = Array(sliderCount)
           .fill(0)
-          .map((_, i) => i + 1);
+          .map((_, i) => i);
```

Comment:
Since we're using an array to store filters in the `IState`, I switched this back to being 0-indexed.